app/src/routes.js

```diff
@@ -228,13 +228,13 @@ export default (store) => {
       component: () => import(`./components/common/FeatureNotPresent`),
     },
     {
-      path: `/sign-in-modal`,
+      path: `/sign-up-email`,
       name: `sign-in-modal`,
       component: require("account/SignInModal").default,
     },
     {
-      path: `/magic-link-sent-modal`,
-      name: `magic-link-sent-modal`,
+      path: `/magic-link`,
+      name: `magic-link-sent`,
```

Comment:
Shouldn't this two better be called the same?

Reply:
Ah ok, because you don't want the word modal in the url. OK

Reply:
i care about URLs - sweat the details!